client/app/scripts/liveblog-edit/directives.js

```diff
@@ -534,15 +542,74 @@ define([
                 }
             };
         }])
-        .directive('freetypeLink', ['$compile', function($compile) {
+        .directive('freetypeText', ['$templateCache', function($templateCache) {
 
             return {
                 restrict: 'E',
-                template: '<input type="url" ng-model="link"/>',
-                controller: function() {
-                },
+                template: $templateCache.get('scripts/liveblog-edit/views/freetype-text.html'),
```

Comment:
in the webpack branch @vladnicoara found a different approach.
this PR is not here to fix that, just to enhance with validation.